Remove guppy and symmetry packages

goldmanm opened the pull request:
I removed the dependencies of `guppy` and `symmetry` from requirements file after noticing that they are not being used, nor can they currently function, in RMG. 

I tested if RMG still functioned by running `conda remove guppy symmetry` and then `make test`. All the tests passed.

rwest commented:
Did you try a job with QMTP  having removed symmetry and ensuring there's not some other version in your path?

mliu49 commented:
We definitely use symmetry for QMTP, so this is probably an indication that it's not being tested properly.

goldmanm commented:
Thanks for the suggestions. I found some old symmetry versions and removed them. `make test` still reported no errors, so I agree with Max that we likely don't have anything testing for this part of QMTP, or the tests are disabled. 

I removed the `symmetry` commit from the pull request. It should now only be removing `guppy`

rwest commented:
I think your local tests without symmetry are still not testing what you think - the Travis tests failed with 7 errors when you removed symmetry:
https://travis-ci.org/ReactionMechanismGenerator/RMG-Py/builds/169015907

```
======================================================================
ERROR: Test that Mocpac getThermoData() works correctly.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/mainTest.py", line 218, in testGetThermoDataMopac
    thermo1 = self.mop1.getThermoData(mol)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/main.py", line 216, in getThermoData
    thermo0 = qm_molecule_calculator.generateThermoData()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 334, in generateThermoData
    self.determinePointGroup()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 413, in determinePointGroup
    self.pointGroup = pgc.calculate()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 133, in calculate
    return self.calculator.calculate();
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 243, in calculate
    output = self.run(command)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 199, in run
    raise Exception('Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.')
Exception: Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.
======================================================================
ERROR: Test that generateThermoData() works correctly for MOPAC PM3
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/mopacTest.py", line 62, in testGenerateThermoData
    self.qmmol1.generateThermoData()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 334, in generateThermoData
    self.determinePointGroup()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 413, in determinePointGroup
    self.pointGroup = pgc.calculate()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 133, in calculate
    return self.calculator.calculate();
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 243, in calculate
    output = self.run(command)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 199, in run
    raise Exception('Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.')
Exception: Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.
======================================================================
ERROR: Test that generateThermoData() can load thermo from the previous MOPAC PM3 run.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/mopacTest.py", line 81, in testLoadThermoData
    self.qmmol1.generateThermoData()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 334, in generateThermoData
    self.determinePointGroup()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 413, in determinePointGroup
    self.pointGroup = pgc.calculate()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 133, in calculate
    return self.calculator.calculate();
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 243, in calculate
    output = self.run(command)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 199, in run
    raise Exception('Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.')
Exception: Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.
======================================================================
ERROR: Test that generateThermoData() works correctly for MOPAC PM6
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/mopacTest.py", line 125, in testGenerateThermoData
    self.qmmol1.generateThermoData()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 334, in generateThermoData
    self.determinePointGroup()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 413, in determinePointGroup
    self.pointGroup = pgc.calculate()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 133, in calculate
    return self.calculator.calculate();
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 243, in calculate
    output = self.run(command)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 199, in run
    raise Exception('Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.')
Exception: Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.
======================================================================
ERROR: Test that generateThermoData() can load thermo from the previous MOPAC PM6 run.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/mopacTest.py", line 144, in testLoadThermoData
    self.qmmol1.generateThermoData()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 334, in generateThermoData
    self.determinePointGroup()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 413, in determinePointGroup
    self.pointGroup = pgc.calculate()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 133, in calculate
    return self.calculator.calculate();
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 243, in calculate
    output = self.run(command)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 199, in run
    raise Exception('Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.')
Exception: Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.
======================================================================
ERROR: Test that generateThermoData() works correctly for MOPAC PM7
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/mopacTest.py", line 189, in testGenerateThermoData
    self.qmmol1.generateThermoData()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 334, in generateThermoData
    self.determinePointGroup()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 413, in determinePointGroup
    self.pointGroup = pgc.calculate()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 133, in calculate
    return self.calculator.calculate();
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 243, in calculate
    output = self.run(command)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 199, in run
    raise Exception('Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.')
Exception: Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.
======================================================================
ERROR: Test that generateThermoData() can load thermo from the previous MOPAC PM7 run.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/mopacTest.py", line 208, in testLoadThermoData
    self.qmmol1.generateThermoData()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 334, in generateThermoData
    self.determinePointGroup()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/molecule.py", line 413, in determinePointGroup
    self.pointGroup = pgc.calculate()
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 133, in calculate
    return self.calculator.calculate();
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 243, in calculate
    output = self.run(command)
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/qm/symmetry.py", line 199, in run
    raise Exception('Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.')
Exception: Running symmetry on the point group calculation has failed.  Please check if symmetry program is installed on your system in RMG-Py/bin/symmetry or on your path.
```


Removing just guppy seems to not be a problem.
https://travis-ci.org/ReactionMechanismGenerator/RMG-Py/builds/169031228
